test/lit.cfg

```diff
@@ -1340,6 +1340,76 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
         '-L%s' % make_path(test_resource_dir, config.target_sdk_name)])
     # The Swift interpreter is not available when targeting Android.
     config.available_features.discard('swift_interpreter')
+elif run_os == 'wasi':
+    tools_directory = pipes.quote(make_path(config.wasi_sdk_path, "bin"))
```

Comment:
What tools are you using from there?  Im afraid that this is going to use a different version of clang/lld/...